### PR TITLE
Add again logging_silent benchmark

### PR DESCRIPTION
**Pull request by haypo**
Revert partially commit b22d38d0f76ed21152e146fec3fa65f76da1372b.

### CODE REVIEW

File: doc/benchmarks.rst [hunk 1]
```diff
@@ -253,6 +252,8 @@ Benchmarks on the ``logging`` module:
 
 * ``logging_format``: Benchmark ``logger.warn(fmt, str)``
 * ``logging_simple``: Benchmark ``logger.warn(msg)``
+* ``logging_silent``: Benchmark ``logger.warn(msg)`` when the message is
```
Comment (pitrou) [inline doc/benchmarks.rst]: Apparently it's `debug()` in the benchmark :-)

File: doc/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Version 0.5.6
+-------------
+
+* Add again the ``pybench`` microbenchmark suite.
```
Comment (pitrou) [inline doc/changelog.rst]: Is this a mistake?

File: doc/benchmarks.rst (hunk 1 above)
Comment (haypo) [inline doc/benchmarks.rst]: right, fixed

File: doc/changelog.rst (hunk 2 above)
Comment (haypo) [inline doc/changelog.rst]: The reverted benchmark was larger, I had to modify the patch. It should now be fixed.

**Comment by haypo** [comment]
I fixed the two commented code. Does it look better?

**Comment by pitrou** [comment]
Looks good to me now!

**Comment by haypo** [comment]
Ok, PR merged. Welcome back microbenchmarks ;-)

**Pull request merged**